src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * add a `workflow_metadata` function
+ * add bundle name suggestion on wrongly overridden templates paths
```

Comment:
this needs to be an item under a to be added `4.2.0` section

Reply:
Updated. Thank you